Applications/Android/kotlin/app/src/main/jni/cifar_dataloader.cpp

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Jihoon Lee <jhoon.it.lee@samsung.com>
```

Comment:
If we use totally duplicated files, how about just linking files ?

Reply:
done. thanks.